charts/argocd-apps/values-integration.yaml

```diff
@@ -497,6 +497,55 @@ govukApplications:
             key: bearer_token
       - name: PUBLISH_TOPICS
         value: "1"
+- name: govuk-replatform-test-app
+  repoName: govuk-replatform-test-app
+  helmValues:
+    sentry:
+      enabled: false
+    uploadAssets:
+      enabled: false
+    appPort: 9292
+    appProbes:
+      startupProbe: &test-probe
+        httpGet:
+          path: /healthcheck/live
+          port: 9292
+        failureThreshold: 10
+        periodSeconds: 1
+        timeoutSeconds: 1
+      livenessProbe:
+        <<: *test-probe
+        failureThreshold: 3
+        periodSeconds: 5
+      readinessProbe:
+        <<: *test-probe
+        failureThreshold: 2
+        periodSeconds: 5
+    ingress:
+      enabled: true
+      annotations:
+        alb.ingress.kubernetes.io/load-balancer-name: govuk-replatform-test-app
+      hosts:
+      - govuk-replatform-test-app.eks.integration.govuk.digital
+    extraEnv:
+      - name: HELM_MESSAGE_1
```

Comment:
Could the app just pick up any Env Var with a prefix? Rather than having a pre-defined number of slots?

Reply:
i.e. prefix like `MESSAGE_`

Reply:
Nit: Not sure about using the name HELM_MESSAGE as the environment variable doesn't actually have anything to do with Helm, which might be confusing for people learning. (Yes, Helm does template the env vars for the app config, but isn't actually effected by their value.)

Reply:
Not sure about picking up any Env Var, I can simplify the prefix to just MESSAGE, reason I put HELM_MESSAGE is that its delivered from the helm charts and not from the test app